canonical-web-and-design/maas-ui
# Update usabilla script

## Done
- Update usabilla to latest script

## QA
- Disable adblockers
- Click "Give feedback" in the footer and put whatever you want
- Log in to the [usabilla app](https://app.usabilla.com/member/#/websites/buttons/) and click on Feedback on the left
- Check that your feedback is there, and click on it
- Check that under "Button" it says "MAAS app 2.8"
- Delete your feedback

## Fixes
Fixes canonical-web-and-design/MAAS-design#903
